Document type predicate expression (#614)
neo4j/docs-cypher: modules/ROOT/pages/values-and-types/working-with-null.adoc

= Working with `null`

In Cypher, `null` is used to represent missing or undefined values.

> All data types in Cypher are nullable, which leads to the xref::syntax/expressions.adoc#type-predicate-expressions-null[type predicate expression] always  returning `true` for `null` values.

Reviewer: ```suggestion
All data types in Cypher are nullable.
This means that xref::syntax/expressions.adoc#type-predicate-expressions-null[type predicate expressions] always return `true` for `null` values.
```
